package.json

```diff
@@ -46,4 +46,4 @@
     "tslib": "^2.4.0",
```

Comment:
No need to commit this.

Reply:
I dont know why this is here tbh, it grabed from my other PR for some reason

Reply:
It's the whitespace at the end of the file. But since we're bumping the version, guess we do need to commit it. ;)